src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -99,7 +99,9 @@ def get(self, request: Request, project) -> Response:
                     match_choices = context["formFields"]["match"]["choices"]
                     copied_context = copy.deepcopy(context)
                     copied_context["formFields"]["match"]["choices"] = [
-                        choice for choice in match_choices if choice[0] != MatchType.IS_IN
+                        choice
+                        for choice in match_choices
+                        if (choice[0] != MatchType.IS_IN or choice[0] != MatchType.NOT_IN)
```

Comment:
Why do we need to strip `is_in` and `not_is_in`? (like it says on line 98)

Reply:
Oh oops - let me update the note

this is now stripping both of them